src/billing/thunks/index.ts

```diff
@@ -11,20 +11,27 @@ import {
   setBillingSettingsStatus,
   setInvoices,
   setInvoicesStatus,
+  setOrgLimits,
+  setOrgLimitsStatus,
 } from 'src/billing/reducers'
 
 // API
 import {
   getAccount as apiGetAccount,
   getBillingInfo as apiGetBillingInfo,
   getBillingNotificationSettings,
+  updateBillingNotificationSettings,
   getInvoices as apiGetInvoices,
+  getOrgsLimits as apiGetOrgLimits,
 } from 'src/billing/api'
 
 // Types
 import {RemoteDataState} from 'src/types'
-import {Invoice} from 'src/types/billing'
+import {BillingNotifySettings, Invoice} from 'src/types/billing'
 
+// TODO(ariel): add error handling here
+// notify() will not work since Dispatch here is based on the passed in dispatch from the local reducer
+// and not from the higher level dispatch from the app.
```

Comment:
Elaborating a bit on this here. I did an audit of the error states and it seems like the two places we aren't actively accounting for errors in the unification work are on the Users page and the Billing page. Both use the reducer pattern over the context pattern making it a little more difficult for us to simply trigger errors from the thunks. We want to refactor these to a context-based pattern in order to prevent issues like these and keep a centralized style rather than a disparate one in different parts of unification